src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/NuGet.PackageManagement.VisualStudio.csproj

```diff
@@ -1,297 +1,61 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
+    <TargetFramework>$(NETFXTargetFramework)</TargetFramework>
     <Shipping>true</Shipping>
-    <IncludeInVSIX>true</IncludeInVSIX>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <ProjectGuid>{306CDDFA-FF0B-4299-930C-9EC6C9308160}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <IncludeInVsix>true</IncludeInVsix>
     <RootNamespace>NuGet.PackageManagement.VisualStudio</RootNamespace>
-    <AssemblyName>NuGet.PackageManagement.VisualStudio</AssemblyName>
-    <FileAlignment>512</FileAlignment>
     <Description>NuGet's Visual Studio functionalities, integrations and utilities.</Description>
-    <ResolveNuGetPackages>true</ResolveNuGetPackages>
-    <TargetFrameworkVersion>$(NETFXTargetFrameworkVersion)</TargetFrameworkVersion>
   </PropertyGroup>
-  <ItemGroup>
-    <Reference Include="System.Runtime.Caching" />
-  </ItemGroup>
-  <ItemGroup>
+
+ <ItemGroup>
     <Reference Include="System" />
     <Reference Include="System.ComponentModel.Composition" />
     <Reference Include="System.Core" />
     <Reference Include="System.IO.Compression" />
+    <Reference Include="System.Runtime.Caching" />
     <Reference Include="System.Runtime.Serialization" />
     <Reference Include="Microsoft.CSharp" />
     <Reference Include="System.Xml.Linq" />
     <Reference Include="WindowsBase" />
-    <Reference Include="System.Management.Automation, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL" />
```

Comment:
How did you migrate both projects? Did you use try-convert or upgrade-assistant?

Is this Reference imported automatically in an SDK-style project?

Reply:
If I compile test project with `dotnet build` this reference is not found.

![image](https://user-images.githubusercontent.com/1192347/204681593-c53ed6f0-61a7-4af4-8379-f14bfe5c8a05.png)

Reply:
Oh, I see, this is why you moved `IPSNuGetProjectContext` to NuGet.Console, so this project can get rid of System.Management.Automation reference.

Reply:
Yep, that's exactly it. 
I migrated everything manually.